assets/src/components/FeatureToolbar.js

```diff
@@ -253,6 +271,31 @@ export default class FeatureToolbar extends HTMLElement {
         return atlasLayouts;
     }
 
+
+    /**
+     * Return the list of children layers for which a feature can be created
+     *
+     * @return array
+     */
+    get editableChildrenLayers() {
+        const editableChildrenLayers = [];
+        lizMap.config?.relations?.[this.layerId]?.some((relation) => {
+
+            // Check if the child layer has insert capabilities
+            let [childFeatureType, childLayerConfig] = lizMap.getLayerConfigById(relation.referencingLayer);
+            if (lizMap.config?.editionLayers?.[childFeatureType]?.capabilities?.createFeature !== "True") {
```

Comment:
I thought you would use the magical hidden existing function to parse a string/bool into a boolean ;-)